src/MiNET/MiNET/Net/MCPE Protocol Documentation.md

```diff
@@ -1841,7 +1841,7 @@ Wiki: [Resource Pack Chunk Data](https://github.com/NiclasOlofsson/MiNET/wiki//P
 |Package ID | string |  |
 |Chunk Index | uint |  |
 |Progress | ulong |  |
-|Length | uint |  |
+|Length | UnsignedVarInt |  |
 |Payload | byte[] | (int) length |
```

Comment:
Maybe you can ditch the length and use `ByteArray` type here, like [this](https://github.com/NiclasOlofsson/MiNET/pull/474/commits/8166d131df4bee4f90e9e1d79f728cddd4a21700#diff-d56386283b680144457b992673874157R1824).

Reply:
Thanks, change it later.